dev_scripts/exportTranslation.js

```diff
@@ -0,0 +1,117 @@
+  fs.readdirSync('././src/localization/').forEach(file => {
+    // Filter to only get JSON files
+    if (file.split('.').pop() === 'json') {
+      const fileName = require(`../src/localization/${file}`);
+      localizationFiles.push(fileName);
+      localizationFiles[index].name = file.replace('.json', '');
+      index += 1;
+    }
+  });
+};
+
+// Gets the language entered by the User (fr, la, gil, tl)
+getSelectedLanguage = () => process.argv[2];
+
+generateCsvData = (mainLanguage, selectedLanguage) => {
+  let valueFound = false;
+  // csvFileData keeps all the data stored in json files
+  let csvData = '';
+  // Goes through all localization files
+  localizationFiles.map(localizationFile => {
+    const mainLanguageKeys = [];
+    const mainLanguageValues = [];
+    const selectedLanguageValues = [];
+    // Iterates over every language
+    for (const [language, mainLanguageKey] of Object.entries(localizationFile)) {
+      switch (language) {
+        case mainLanguage:
+          // Saves all mainLanguageValues
+          for (let [mainLanguageKeyAgain, mainLanguageValue] of Object.entries(mainLanguageKey)) {
+            mainLanguageKeys.push(mainLanguageKeyAgain);
+            mainLanguageValue = mainLanguageValue
+              .replace(/\n/gi, '{nextLine}')
+              .replace(/[\"]/gi, '{emptySpace}');
+            mainLanguageValues.push(mainLanguageValue);
+          }
+          break;
+        case selectedLanguage:
+          // Uses mainLanguageKeys array to iterates over the selectedLanguage
+          for (selectedLanguageKey of mainLanguageKeys) {
+            valueFound = false;
+            // Saves all the selectedLanguage data in selectedLanguageValues
+            for (let [selectedLanguageKeyAgain, selectedLanguageValue] of Object.entries(
+              mainLanguageKey
+            )) {
+              if (selectedLanguageKey === selectedLanguageKeyAgain && !valueFound) {
+                selectedLanguageValue = selectedLanguageValue
+                  .replace(/\n/gi, '{nextLine}')
+                  .replace(/[\"]/gi, '{emptySpace}');
+                selectedLanguageValues.push(selectedLanguageValue);
+                // Stops searching the selectedLanguageKey if found
+                valueFound = true;
+              }
+            }
+            if (!valueFound) {
+              // Key an empty value when SelectedLanguageValue is not found
+              selectedLanguageValues.push('');
+              valueFound = false;
+            }
+          }
+          break;
+      }
+    }
+    // Updates (Adds more data) to csvData
+    mainLanguageKeys.map((mainLanguageKey, mainLanguageKeyIndex) => {
+      csvData = `${csvData} ${localizationFile.name},${mainLanguageKey},"${mainLanguageValues[mainLanguageKeyIndex]}","${selectedLanguageValues[mainLanguageKeyIndex]}"\n`;
+    });
+  });
+  return csvData;
+};
+
+writeCsvFile = (selectedLanguage, csvFileData) => {
+  fs.writeFile(
```

Comment:
would rather you use writeFileSync within try catch block (want to avoid using callbacks as much as possible).

Also translations directory doesn't exist, so getting constant error when running export script.
Need to commit empty directory: https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

Reply:
Thanks for this. I just added the folder `src/localization/translations`